backend/app/db/models/geography.py

```diff
@@ -25,7 +25,7 @@ class GeoStatistics(Base):
     __tablename__ = "geo_statistics"
 
     id = sa.Column(sa.SmallInteger(), primary_key=True)
-    name = sa.Column(sa.Text(), unique=True)
+    name = sa.Column(sa.Text(), nullable=False, unique=True)
```

Comment:
I can't see this change reflected in the migrations :thinking:

Reply:
I believe this is the default